packages/features/ee/organizations/components/AddNewTeamsForm.tsx

```diff
@@ -0,0 +1,97 @@
+import { ArrowRight } from "lucide-react";
+import { useRouter } from "next/router";
+import { useState } from "react";
+import { z } from "zod";
+
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { trpc } from "@calcom/trpc/react";
+import { Button, showToast, TextField } from "@calcom/ui";
+import { Plus, X } from "@calcom/ui/components/icon";
+
+const querySchema = z.object({
+  id: z.string().transform((val) => parseInt(val)),
+});
+
+export const AddNewTeamsForm = () => {
+  const { t } = useLocale();
+  const router = useRouter();
+  const { id: orgId } = querySchema.parse(router.query);
+  const [counter, setCounter] = useState(1);
+
+  const [inputValues, setInputValues] = useState<string[]>([""]);
+
+  const handleCounterIncrease = () => {
+    setCounter((prevCounter) => prevCounter + 1);
+    setInputValues((prevInputValues) => [...prevInputValues, ""]);
+  };
+
+  const handleInputChange = (index: number, value: string) => {
+    const newInputValues = [...inputValues];
+    newInputValues[index] = value;
+    setInputValues(newInputValues);
+  };
+
+  const handleRemoveInput = (index: number) => {
+    const newInputValues = [...inputValues];
+    newInputValues.splice(index, 1);
+    setInputValues(newInputValues);
+    setCounter((prevCounter) => prevCounter - 1);
+  };
+
+  const createTeamsMutation = trpc.viewer.organizations.createTeams.useMutation({
+    async onSuccess() {
+      router.push(`/getting-started`);
+    },
+    onError: (error) => {
+      showToast(error.message, "error");
+    },
+  });
+
+  return (
+    <>
+      {Array.from({ length: counter }, (_, index) => (
+        <div className="relative" key={index}>
+          <TextField
+            key={index}
+            value={inputValues[index]}
+            onChange={(e) => handleInputChange(index, e.target.value)}
+          />
+          {index > 0 && (
+            <Button
+              color="minimal"
+              className="absolute top-2 right-2 m-0 flex h-auto p-0"
+              onClick={() => handleRemoveInput(index)}>
+              <X className="bg-subtle text h-5 w-5 rounded-full p-1" />
+            </Button>
+          )}
```

Comment:
Should we turn this into a textfield with addonsuffix? 

To keep this design close to the other places in the app that do the same as this?

Reply:
Done